core/src/syn/v1/stmt/define/table.rs

```diff
@@ -23,13 +29,15 @@ pub fn table(i: &str) -> IResult<&str, DefineTableStatement> {
 	let (i, name) = cut(ident)(i)?;
 	let (i, opts) = many0(table_opts)(i)?;
 	let (i, _) = expected(
-		"DROP, SCHEMALESS, SCHEMAFUL(L), VIEW, CHANGEFEED, PERMISSIONS, or COMMENT",
+		"TYPE, RELATION, DROP, SCHEMALESS, SCHEMAFUL(L), VIEW, CHANGEFEED, PERMISSIONS, or COMMENT",
```

Comment:
`RELATION` on it's own is not a clause right?
```suggestion
		"TYPE, DROP, SCHEMALESS, SCHEMAFUL(L), VIEW, CHANGEFEED, PERMISSIONS, or COMMENT",
```

Reply:
both
`DEFINE TABLE likes RELATION;`
and
`DEFINE TABLE likes TYPE RELATION;`
are currently supported